client/a8c-for-agencies/data/purchases/use-fetch-licenses.ts

```diff
@@ -0,0 +1,63 @@
+import config from '@automattic/calypso-config';
+import { useQuery } from '@tanstack/react-query';
+import { LICENSES_PER_PAGE } from 'calypso/a8c-for-agencies/sections/purchases/lib/constants';
+import {
+	LicenseFilter,
+	LicenseSortDirection,
+	LicenseSortField,
+} from 'calypso/jetpack-cloud/sections/partner-portal/types';
+
+export default function useFetchLicenses(
+	filter: LicenseFilter,
+	search: string,
+	sortField: LicenseSortField,
+	sortDirection: LicenseSortDirection,
+	page: number
+) {
+	const showDummyData = config.isEnabled( 'a4a/mock-api-data' );
```

Comment:
[non-blocker] Since we don't have a working API to protect, let's skip adding the `a4a/mock-api-data` feature.

Reply:
This will be used to test the feature in horizon and staging later.

Reply:
make sense